packages/manager/modules/pci/src/projects/project/additional-ips/additional-ips.module.js

```diff
@@ -18,15 +19,16 @@ angular
   .module(moduleName, [
     'ui.router',
     'pascalprecht.translate',
-    additionalIp,
+    failoverIps,
+    floatingIps,
     imports,
     instancesModule,
     onboarding,
     order,
   ])
   .config(routing)
   .component('pciProjectAdditionalIps', component)
-  .service('additionalIpService', service)
+  .service('AdditionalIpService', service)
```

Comment:
**nitpick:** We tend to prefix service name according to its module location.

Reply:
Done.